sgkit/stats/popgen.py

```diff
@@ -554,8 +554,10 @@ def Tajimas_D(
 
     ac = ds[variant_allele_count]
 
-    # count segregating
-    S = ((ac > 0).sum(axis=1) > 1).sum()
+    # count segregating. Note that this uses the definition in tskit,
+    # which is the number of alleles - 1. In the biallelic case this
+    # gives us the number of non-monomorphic sites.
+    S = ((ac > 0).sum(axis=1) - 1).sum()
```

Comment:
It looks like windowing has not been implemented for Tajima's D yet. The segregating sites variable (`S`) needs to be calculated per-window (using `window_statistic`) rather than having a total sum calculated (which assumes a single window).